transmogrifier/sources/dspace_dim.py

```diff
@@ -0,0 +1,243 @@
+
+        # notes
+        descriptions = xml.find_all("dim:field", element="description")
+        for description in [
+            d
+            for d in descriptions
+            if d.get("qualifier")
+            not in [
+                "abstract",
+                "provenance",
+                "sponsorship",
+            ]
+            and d.string
+        ]:
+            n = Note(value=[description.string], kind=description.get("qualifier"))
+            kwargs.setdefault("notes", []).append(n)
+
+        # publication_information
+        for publisher in [
+            p for p in xml.find_all("dim:field", element="publisher") if p.string
+        ]:
+            kwargs.setdefault("publication_information", []).append(publisher.string)
+
+        # related_items, uses related_identifiers retrieved for identifiers
+        for related_item in [
+            r for r in xml.find_all("dim:field", element="relation") if r.string
+        ]:
+            if related_item.get("qualifier") == "uri":
+                ri = RelatedItem(uri=related_item.string)
+            else:
+                ri = RelatedItem(
+                    description=related_item.string,
+                    relationship=related_item.get("qualifier"),
+                )
+            kwargs.setdefault("related_items", []).append(ri)
+
+        # rights
+        for rights in [
+            r for r in xml.find_all("dim:field", element="rights") if r.string
+        ]:
+            if rights.get("qualifier") == "uri":
+                rg = Rights(uri=rights.string)
+            else:
+                rg = Rights(description=rights.string, kind=rights.get("qualifier"))
+            kwargs.setdefault("rights", []).append(rg)
+
+        # subjects
+        subjects_dict: Dict[str, List[str]] = {}
+        for subject in [
+            s for s in xml.find_all("dim:field", element="subject") if s.string
+        ]:
+            if subject.get("qualifier") is None:
+                subjects_dict.setdefault("Subject scheme not provided", []).append(
+                    subject.string
+                )
+            else:
+                subjects_dict.setdefault(subject.attrs["qualifier"], []).append(
+                    subject.string
+                )
+        for key, value in subjects_dict.items():
+            s = Subject(value=value, kind=key)
+            kwargs.setdefault("subjects", []).append(s)
+
+        # summary, uses description list retrieved for notes field
+        for description in [
+            d for d in descriptions if d.get("qualifier") == "abstract" and d.string
+        ]:
+            kwargs.setdefault("summary", []).append(description.string)
+
+        # citation, uses identifiers list retrieved for identifiers field
+        citation = ""
+        for identifier in [
+            i for i in identifiers if i.get("qualifier") == "citation" and i.string
+        ]:
+            citation = identifier.string
```

Comment:
This is going to result in `citation` being set to the last citation value if there are multiple (which there shouldn't be, but...). Since TIMDEX only allows one citation string, I would probably go with the first one? e.g. `xml.find("dim:identifier", qualifier="citation")`.